### ISSUE TITLE
You were banned on Scratch? (/

### ISSUE THREAD

**Issue by windowsmas** [PPPDUD/PlatformDT#1]
This may be offtopic.
-I'm jellyfishfisher56 (you know me on Scratch, right?)
-You said in your Ocular status you were blocked.
-why were you blocked?

**Comment by PPPDUD** [comment]
Yes. Are you a Scratcher?

**Comment by windowsmas** [comment]
@PPPDUD Read my edited first comment.

**Comment by PPPDUD** [comment]
> @PPPDUD Read my edited first comment.

Okay. I'll answer your questions:
I was banned for ban evading on SUI Lookup (which was banned for quickly getting alerts)
Yes, I remember you
I did indeed state that

**Comment by windowsmas** [comment]
Use Contact Us - you were banned for no reason on sui-lookup.

**Comment by PPPDUD** [comment]
I've tried, but they appear to intentionally ignore my ban appeals.

**Comment by windowsmas** [comment]
scratch team support go brr

**Comment by windowsmas** [comment]
make an alt (yes, i know its ban evading) but dont mention you are pppdud
make the name whatever you like

**Comment by PPPDUD** [comment]
Nope, not ban evading. If I get IP banned, then it will be illegal for me
to continue.

**Comment by windowsmas** [comment]
what should you do then?
any ideas?
you could leave scratch i guess but that's urr..

**Comment by PPPDUD** [comment]
I might consider it if nothing else works. Visit my website at
mojavesoft.net for other cool things that the ST can't stop.

**Comment by windowsmas** [comment]
You can use scratchquickappeal.tools.jb.jbdev.oneclick.freeserverhosting.nw which prioritises it which is reserved for Scratch Team members.
There's even users and usernames. Mine is lewisdavidsons.

**Comment by PPPDUD** [comment]
I can't seem to navigate to that URL. Have you mistyped something?

**Comment by windowsmas** [comment]
projects.iamjb.jb.github.io/scratch-quick-appeals-v-1.2-thanks-for-the-pull-request-niraqu-and-windowsmas

**Comment by PPPDUD** [comment]
404 error

**Comment by windowsmas** [comment]
well then
add #'{ "scratchteam" : "true"}' to the end of the subject of your appeal

**Comment by PPPDUD** [comment]
Can you ask someone like PizzaPizza74 to prove this isn't a bunch of
baloney and propaganda because I use a Mac?

**Comment by windowsmas** [comment]
What do you mean? If you use a Mac, it won't work because iMail represents tags as your email sections.
Send the appeal online, on something like Gmail or Outlook.

**Comment by PPPDUD** [comment]
I have sent it online with Gmail. Still no response.

**Comment by windowsmas** [comment]
Do you really think that the Scratch Team will respond in 22 seconds? (that's the time between our comments)

**Comment by PPPDUD** [comment]
No. I sent it a day ago.

**Comment by windowsmas** [comment]
A day ago, you didn't know that you needed to put #{ "scratchteam" : "true"}' on the end of the subject, therefore it's just the same as any appeal.

**Comment by PPPDUD** [comment]
I still don't trust that adding simple JSON to appeals actually makes this
work faster, and a reliable non-social media source of this info would be
nice.

**Comment by windowsmas** [comment]
what are you going to do about your scratch account then

**Comment by PPPDUD** [comment]
No idea. I still don't trust that simple JSON will help without reliable
proof, though.

**Comment by windowsmas** [comment]
You're still banned

**Comment by PPPDUD** [comment]
The Scratch Team has said that my account won't be unbanned.
